### ISSUE TITLE
Multiple models can be uploaded in a demo.

### ISSUE THREAD

**Issue by MrXin95** [AR-js-org/studio#99]
I found that in the location-based section, you can add multiple locations, but only one file can be uploaded, whichmake no scene. I think that every time you add a location, you can upload one more file, as well as for images-based AR.

**Comment by gciandro13** [comment]
Hi @MrXin95  we will take into account this suggestion ;)

**Comment by nicolocarpignoli** [comment]
@MrXin95 Yes, that will be a nice addition. It will require a lot of work from the UI/UX side, but it's definitely valuable.

**Comment by hilarytsai** [comment]
Seriously, I don't even think it's usable if you can't have more than one asset.... I'm having to switch to a different solution now because I can't build my scene with only one asset.
